src/index/IndexImpl.cpp

```diff
@@ -400,9 +423,12 @@ void IndexImpl::createFromFiles(
         buildOspWithPatterns(std::move(patternOutput.value()),
                              *indexBuilderData.sorter_.internalTriplesPso_);
     createInternalPsoAndPosAndSetMetadata();
-    createThirdPermutationPair(NumColumnsIndexBuilding + 2,
 
-                               thirdSorterPtr->template getSortedBlocks<0>());
+    auto thirdSorterBlocks{ad_utility::InputRangeTypeErased{
```

Comment:
Same TODO here (once `getSortedBlocks` is not a coroutine anymore, we can address this.